sections/collapsible-content.liquid

```diff
@@ -0,0 +1,383 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content collapsible-{{ section.settings.layout | split: '_' | first }}-layout color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content__wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div class="collapsible-content__header" style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption | escape }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="collapsible-content__heading">{{ section.settings.heading | escape }}</h2>
+        {%- else -%}
+          <h2 class="visually-hidden">{{ 'accessibility.collapsible_content_title' | t }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet collapsible-content__grid{% if section.settings.desktop_layout == 'text_first' %} collapsible-content__grid--reverse{% endif %}">
+        {%- if section.settings.image != blank -%}
+          <div class="grid__item grid__item--image">
+            <div class="collapsible-content__media collapsible-content__media--{{ section.settings.image_ratio }} media"
+              {% if section.settings.image_ratio == 'adapt' and section.settings.image != blank %} style="padding-bottom: {{ 1 | divided_by: section.settings.image.aspect_ratio | times: 100 }}%;"{% endif %}
```

Comment:
This is inside a `{%- if section.settings.image != blank -%}`, so checking for a blank image again would be redundant.
```suggestion
              {% if section.settings.image_ratio == 'adapt' %} style="padding-bottom: {{ 1 | divided_by: section.settings.image.aspect_ratio | times: 100 }}%;"{% endif %}
```

Reply:
The eyes of an eagle.